egs/formosa/s5/run.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
+#
+# Copyright 2018, Yuan-Fu Liao, National Taipei University of Technology, yfliao@mail.ntut.edu.tw
+#
+# Before you run this recipe, please apply, download and put or make a link of the corpus under this folder (folder name: "NER-Trs-Vol1").
+# For more detail, please check:
+# 1. Formosa Speech in the Wild (FSW) project (https://sites.google.com/speech.ntut.edu.tw/fsw/home/corpus)
+# 2. Formosa Speech Recognition Challenge (FSW) 2018 (https://sites.google.com/speech.ntut.edu.tw/fsw/home/challenge)
+stage=-2
+num_jobs=20
+
+# shell options
+set -eo pipefail
+
+. ./cmd.sh
+. ./utils/parse_options.sh
+
+# configure number of jobs running in parallel, you should adjust these numbers according to your machines
+# data preparation
+if [ $stage -le -2 ]; then
+
+  # Lexicon Preparation,
+  echo "$0: Lexicon Preparation"
+  local/prepare_dict.sh || exit 1;
+
+  # Data Preparation
+  echo "$0: Data Preparation"
+  local/prepare_data.sh || exit 1;
+
+  # Phone Sets, questions, L compilation
+  echo "$0: Phone Sets, questions, L compilation Preparation"
+  rm -rf data/lang
+  utils/prepare_lang.sh --position-dependent-phones false data/local/dict \
+      "<SIL>" data/local/lang data/lang || exit 1;
+
+  # LM training
+  echo "$0: LM training"
+  rm -rf data/local/lm/3gram-mincount
+  local/train_lms.sh || exit 1;
+
+  # G compilation, check LG composition
+  echo "$0: G compilation, check LG composition"
+  utils/format_lm.sh data/lang data/local/lm/3gram-mincount/lm_unpruned.gz \
+      data/local/dict/lexicon.txt data/lang_test || exit 1;
+
+fi
+
+# Now make MFCC plus pitch features.
+# mfccdir should be some place with a largish disk where you
+# want to store MFCC features.
+mfccdir=mfcc
+
+# mfcc
+if [ $stage -le -1 ]; then
+
+  echo "$0: making mfccs"
+  for x in train test; do
```

Comment:
please merge the run.sh and eval.sh into a single script.
And reduce the amount of vertical whitespace in this script, e.g. at the beginning and end of these if blocks.